tests/register/test_canopen_register.py

```diff
@@ -1,12 +1,73 @@
 import pytest
 
 from ingenialink.canopen.dictionary import CanopenDictionary
-from ingenialink.canopen.register import CanopenRegister
-from ingenialink.register import REG_DTYPE, REG_ACCESS, REG_PHY
+from ingenialink.canopen.register import CanopenRegister, REG_DTYPE, REG_ACCESS, REG_PHY
+from ingenialink.utils._utils import *
 
 
+@pytest.mark.develop
 @pytest.mark.canopen
```

Comment:
This test does not depends on a canopen connection, you can remove this decorator